### PR TITLE
Pill Bottles can only store pills now

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

Made pill bottles only able to store pills.
## Why / Balance

1. Syringes, bottles and paper, which all could fit in pill bottles, shouldn't logically fit. The pills in game are absolutely tiny, syringes are quite long.
2. Bottles being probably the worst example. A pill bottle has 10 slots. Each of these can store a bottle, holding 30u per slot, or 300u in total, all coming down to a 1x1 slot in your inventory. Busted.
In general, pill bottles are one of the most meta storage items, allowing for insane storage. A lot of these got eliminated when GridInv got implemented, this is just clearing up a remaining example. 
## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Pill bottles can now only store pills.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/chemistry.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Specific/chemistry.yml:531]:
```suggestion
      tags:
      - Pill
```

**Review by slarticodefast — APPROVED** [review]
Thanks for the fix!

**Comment by BlitzTheSquishy** [comment]
https://github.com/space-wizards/space-station-14/pull/28364

literally the same PR
but yours got approved, i am in agony.

**Comment by SlamBamActionman** [comment]
> #28364
> 
> literally the same PR but yours got approved, i am in agony.

Some PRs may go through and some may not depending on review procedure and how PRs are judged by maintainers based on the current state of the game (iirc your PR was made during the era all PRs needed to have a 3 day discussion thread and a 75% approval vote among maintainers to go through, while the merge requirements now instead are 2 approvals from maintainers with no explicit disapproval). To expand on the maintainer discussion this PR had, we're still debating what the best way to do the "small items in small containers" feature (some of the sentiments expressed in your PR are still present) and how a design doc should be written about inventories in general, so it's a fairly contentious topic still!

**Comment by BlitzTheSquishy** [comment]
> Member
(I know things change it was mostly as a joke)

**Comment by BlitzTheSquishy** [comment]
> > #28364
> > literally the same PR but yours got approved, i am in agony.
> 
> Some PRs may go through and some may not depending on review procedure and how PRs are judged by maintainers based on the current state of the game (iirc your PR was made during the era all PRs needed to have a 3 day discussion thread and a 75% approval vote among maintainers to go through, while the merge requirements now instead are 2 approvals from maintainers with no explicit disapproval). To expand on the maintainer discussion this PR had, we're still debating what the best way to do the "small items in small containers" feature (some of the sentiments expressed in your PR are still present) and how a design doc should be written about inventories in general, so it's a fairly contentious topic still!

I know it was just, mostly a joke

**Pull request merged**
